docs/machine-learning/resources/tasks.md

```diff
@@ -2,12 +2,14 @@
 title: Machine learning tasks
 description: Explore the different machine learning tasks and associated tasks that are supported in ML.NET.
 ms.custom: seodec18
-ms.date: 04/23/2019
+ms.date: 12/23/2019
 author: natke
 ---
 # Machine learning tasks in ML.NET
 
-When building a machine learning model, you first need to define what you are hoping to achieve with your data. This allows you to choose the right machine learning task for your situation. The following list describes the different machine learning tasks that you can choose from and some common use cases. For more information about choosing the task that is appropriate for your scenario, see [Algorithms](../how-to-choose-an-ml-net-algorithm.md).
+A machine learning task is determined by the type of problem you are trying to solve and the data you have.
+
+The following list describes the different machine learning tasks that you can choose from and some common use cases. For more information about choosing the task that is appropriate for your scenario, see [Algorithms](../how-to-choose-an-ml-net-algorithm.md).
```

Comment:
In this paragraph, the wording in the last sentence may be confusing. If I am in the Tasks doc, why do I need to go to the Algorithms doc to help me choose a task?

Reply:
Yes I agree. I've clarified the text. There is also an item on the backlog to completely refactor this doc to better explain machine learning tasks.